### PR TITLE
New package: schollz.croc version 9.6.2

### LINKED ISSUE #30231: [Package Request]: schollz.croc
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: zack.scholl
Publisher Url: https://github.com/schollz/croc
Package Name: croc
Description: 
README.md

croc
Version Coverage Build Status

This project is supported by:

croc is a tool that allows any two computers to simply and securely transfer files and folders. AFAIK, croc is the only CLI file-transfer tool that does all of the following:

    allows any two computers to transfer data (using a relay)
    provides end-to-end encryption (using PAKE)
    enables easy cross-platform transfers (Windows, Linux, Mac)
    allows multiple file transfers
    allows resuming transfers that are interrupted
    local server or port-forwarding not needed
    ipv6-first with ipv4 fallback
    can use proxy, like tor

For more information about croc, see my blog post.
Package Version: 9.4.2
Installer URL: https://github.com/schollz/croc/releases/tag/v9.4.2

Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by sitiom**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

* Resolves #30231

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94796)

### CODE REVIEW

**Comment by sitiom** [comment]
Winget 1.4

**Comment by Trenly** [comment]
@msftbot dependencies

**Comment by sitiom** [comment]
Dependencies?

File: manifests/s/schollz/croc/9.6.2/schollz.croc.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    PortableCommandAlias: croc
+  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM64.zip
+  InstallerSha256: 9DB19070F24FA0BBD6DCA56F2CE72AD9941ECEA15215BAE967D35EC13B8B0FE8
+- Architecture: arm
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: croc.exe
+    PortableCommandAlias: croc
+  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM.zip
+  InstallerSha256: 8D55CE05005928C75DC410C87453E3B2A0092632F314E45CF420FC0127DB3552
```
Comment (KaranKad) [inline manifests/s/schollz/croc/9.6.2/schollz.croc.installer.yaml:36]:
```suggestion
NestedInstallerType: portable
NestedInstallerFiles:
- RelativeFilePath: croc.exe
  PortableCommandAlias: croc
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-64bit.zip
  InstallerSha256: C46C2D6D2B92CCA3347FAB2D63544819BAAE9611460C67BAF78FFE399C7555AB
- Architecture: x86
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-32bit.zip
  InstallerSha256: 7CD7AB8681063B9CDA00A44E5FC57BBA9757911FC595C447286F90C6CAA4176A
- Architecture: arm
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM.zip
  InstallerSha256: 8D55CE05005928C75DC410C87453E3B2A0092632F314E45CF420FC0127DB3552
- Architecture: arm64
  InstallerUrl: https://github.com/schollz/croc/releases/download/v9.6.2/croc_9.6.2_Windows-ARM64.zip
  InstallerSha256: 9DB19070F24FA0BBD6DCA56F2CE72AD9941ECEA15215BAE967D35EC13B8B0FE8
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/s/schollz/croc/9.6.2/schollz.croc.installer.yaml (hunk 1 above)
Comment (sitiom) [reply manifests/s/schollz/croc/9.6.2/schollz.croc.installer.yaml:36]: YamlCreate doesn't seems to do this, would be a great feature.

**Review by KaranKad — APPROVED** [review]

**Review by KaranKad — APPROVED** [review]

**Review by KaranKad — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**